content/developer/vanilla-cli/build-process.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Build Process
+tags:
+- Developers
+- CLI
+- Build
+- Tools
+- Processes
+category: developer
+menu:
+  developer:
+    parent: cli
+---
+
+The CLI contains multiple build processes. All new projects should use the latest version.
+
+## 1.0
+
+This is the primary build process. It is based on the build tool [gulp](http://gulpjs.com/) It builds stylesheets using [node-sass](https://github.com/sass/node-sass), bundles javascript with [Webpack](https://github.com/webpack/webpack) and [babel](https://babeljs.io/).
+
+### Folder Structure
+
+Source files are always found in one of two places - the `src` directory or the `node_modules` directory. Node modules can be installed from [npm](https://www.npmjs.com/) with [yarn](https://yarnpkg.com/en/).
+
+#### Sass
+
+This build process works with the [Sass preprocessor](http://sass-lang.com/). Source files should be placed in the `src/scss` directory. Any file ending in `*.scss` that does not begin with an underscore `_` is considered a source file. Filenames beginning with an underscore such as `_partial.scss` signify a partial file to sass. These do not get built on there own and must use the `@import` directive to include them. See [The Sass language guide](http://sass-lang.com/guide) for more details.
+
+#### Javascript
+
+The javascript entry point is currently always found at `src/js/index.js`. It does not currently support multiple javascript entry files.
+
+#### Images
+
+Image files can be placed in the `src/images` folder. Each image is its own entry. Images in the src folder will be minified using [ImageMin](https://github.com/imagemin/imagemin) before they are copied to the output directory.
+
+#### Output
```

Comment:
Empty section?

Reply:
It was supposed to be an h3. I've adjusted the page structure and it's now an h2 with h3's underneath it.